scripts/detect_changes.sh

```diff
@@ -2,9 +2,7 @@
 
 set -e
 
-# Note: the .* here can be removed when https://github.com/nodatime/nodatime/issues/502 has been fixed
-RELEASE=`wget -q -O- http://nodatime.org/tzdb/latest.txt | sed -r 's/.*tzdb([a-z0-d]+)\.nzd.*/\1/g'`
-
+RELEASE=`wget -q -O - http://www.iana.org/time-zones | sed -rn 's/.*\/time-zones\/repository\/releases\/tzdata([A-Za-z0-9]+).*/\1/p'`
 HTML=`wget -q -O- http://nodatime.github.io/tzvalidate/`
```

Comment:
Done. Didn't think github pages supported https, but apparently that's only in terms of custom domains.